src/XrdHttp/XrdHttpProtocol.cc

```diff
@@ -2750,6 +2765,25 @@ int XrdHttpProtocol::xsecxtractor(XrdOucStream& Config) {
   return 0;
 }
 
+int XrdHttpProtocol::xcors(XrdOucStream& Config) {
+  char * val;
+  // Get the path
+  //
+  val = Config.GetWord();
+  if (!val || !val[0]) {
+    eDest.Emsg("Config", "No CORS plugin specified.");
+    return 1;
+  } else {
+    char libName[4096];
+    strlcpy(libName, val, sizeof(libName));
+    libName[sizeof(libName) - 1] = '\0';
+    if (LoadCorsHandler(&eDest, libName)) {
```

Comment:
Please move the load to the cors configuration method called after parsing the config file as it eleongs there not here.

Reply:
OK I inspired from the xsecxtractor which does this load:
```cpp
// Try to load the plugin (if available) that extracts info from the
    // user cert/proxy
    if (LoadSecXtractor(&eDest, libName, libParms)) {
      return 1;
    }
```

It avoided to have to declare a string to store the libName. However no problem, will do :)